.docker/ci/Dockerfile

```
@@ -31,7 +31,7 @@ RUN \
         sudo \
         # Preferred build tools
         python-catkin-tools \
-        ccache && \
+        clang ccache && \
```

please separate this onto new line to keep consistent format and make git changes easier to track
```suggestion
        clang && \
        ccache && \
```